.gitignore

```diff
@@ -1,2 +1,4 @@
 coverage_unit.txt
 jupyter-apis
+meow-old
+crud-web-apps
```

Comment:
If not needed, please remove

Reply:
no the meow is necessary

Reply:
@Jose-Matsuda Is it really or do you just not want the meow to go away?